web/src/components/TaskProgressTagLookup.tsx

```diff
@@ -1,22 +1,28 @@
 import { Tag } from "@/components/njwds-extended/Tag";
-import Config from "@businessnjgovnavigator/content/fieldConfig/config.json";
+import { useConfig } from "@/lib/data-hooks/useConfig";
```

Comment:
Changes in this file are "significant"

Reply:
If we wanted to keep the same pattern here, we could use `getMergedConfig` instead